packages/graphql_codegen/test/assets/add_typenames/schema.graphql.dart

```diff
@@ -381,11 +455,35 @@ class Query$Q$docsWihtoutTypename {
 
 extension UtilityExtension$Query$Q$docsWihtoutTypename
     on Query$Q$docsWihtoutTypename {
-  Query$Q$docsWihtoutTypename copyWith(
-          {String? Function()? title, String? $__typename}) =>
-      Query$Q$docsWihtoutTypename(
-          title: title == null ? this.title : title(),
-          $__typename: $__typename == null ? this.$__typename : $__typename);
+  CopyWith$Query$Q$docsWihtoutTypename<Query$Q$docsWihtoutTypename>
+      get copyWith => CopyWith$Query$Q$docsWihtoutTypename(this, (i) => i);
+}
+
+abstract class CopyWith$Query$Q$docsWihtoutTypename<TRes> {
+  factory CopyWith$Query$Q$docsWihtoutTypename(
+          Query$Q$docsWihtoutTypename instance,
+          TRes Function(Query$Q$docsWihtoutTypename) then) =
+      _CopyWithImpl$Query$Q$docsWihtoutTypename;
+
+  TRes call({String? title, String? $__typename});
```

Comment:
Handle the case better where the field isn't nullable. In this case we need to treat `null` and `undefined` in the same way.